elements/pfe-card/src/pfe-card.scss

```diff
@@ -15,29 +15,29 @@ $variables: (
   Padding: #{pfe-local(PaddingTop)} #{pfe-local(PaddingRight)} #{pfe-local(PaddingBottom)} #{pfe-local(PaddingLeft)},
 
   //-- Border settings
-  BorderRadius: pfe-var(surface--border-radius),
+  BorderRadius: #{pfe-var(surface--border-radius)},
 
   //-- Border variable encompasses width, style, and color
-  Border: #{pfe-local(BorderWeight, 0)} #{pfe-local(BorderStyle, solid)} #{pfe-local(BorderColor, transparent)},
+  Border: #{pfe-local(BorderWidth, 0)} #{pfe-local(BorderStyle, solid)} #{pfe-local(BorderColor, transparent)},
+  BorderTop: #{pfe-local(Border)},
 
   //-- Color properties
-  BackgroundColor: pfe-color(surface--base),
+  BackgroundColor: #{pfe-color(surface--base)},
   BackgroundPosition: center center,
 
   spacing: pfe-var(container-spacer),
 
   // Internal spacing; elements inside the card (header, body, footer regions)
-  spacing--vertical: pfe-local(spacing),
+  spacing--vertical: #{pfe-local(spacing)},
```

Comment:
I'm confused about how works with the existing `spacing` var, it seems to just override it. Could it maybe act as a spacing unit only on the header region (and thus change it's name to `spacing--header`?

Reply:
It's just a hook for a user to optionally use.  If they set spacing, it will use that, but if you want to specifically override the internally spacing, you can do that here and it will use that instead.